modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/logit/option/StoreOptions.java

```diff
@@ -27,13 +27,13 @@ public class StoreOptions {
 
     private static final String storagePath                   = "localLog";
 
-    // Default is 0.5G
-    private int                 segmentFileSize               = 1024 * 1024 * 512;
+    // Default is 64Mb.
```

Comment:
There is probably a reason for the default to differ from the original default. Could it be put in this comment?

Reply:
I made it equal to defaults in Ignite 2. I'll mention that in the comment as well